.cloudbees/workflows/workflow.yml

```diff
@@ -9,19 +9,41 @@ on:
       - "*"
 
 jobs:
+  build:
+    steps:
+      - name: Checkout
+        uses: cloudbees-io/checkout@v1
+      - name: Self Test
+        uses: docker://golang:1.21.7
+        run: |
+          make format sync check-git-status test
+      - id: dockerconfig
+        name: Configure container registry credentials
+        uses: cloudbees-io/configure-oci-credentials@v0
+        with:
+          registry: ${{ vars.STAGING_DOCKER_REGISTRY }}
```

Comment:
Not sure if it good idea to pin public repo actions to staging specific variables.

Reply:
A different environment can use a different value. This way I am not hard coding

Reply:
Ok we an proceed - at least do not see better solution for now.